### ISSUE TITLE
Add a AST explorer in Publicodes studio

### ISSUE THREAD

**Issue by mquandalle** [betagouv/publicodes#160]
Like in the Svelte REPL https://svelte.dev/repl/hello-world
You can hover AST nodes and it highlights the matching code.

**Comment by johangirod** [comment]
Yes ! And it will lead to better error messages too (as the info about the col and line number would have to be added to the ASTNode)

**Comment by mquandalle** [comment]
Here is a fork of Svelte REPL for a custom made publicodes parser: https://publicodes-compiled.vercel.app (super early)
